app/serializers/basic_user_serializer.rb

```diff
@@ -1,5 +1,9 @@
 class BasicUserSerializer < ApplicationSerializer
-  attributes :id, :username, :avatar_template
+  attributes :id, :username, :name, :avatar_template
+
+  def name
+    Hash === user ? user[:name] : user.try(:name)
```

Comment:
I noted that we have no existing specs for the serializer but it'll be good to have this tested so that we can document this  behavior in some way and prevent it from regressing.

Reply:
Done.